src/libasr/runtime/lfortran_intrinsics.c

```diff
@@ -854,7 +854,7 @@ LFORTRAN_API char* _lfortran_str_copy(char* s, int32_t idx1, int32_t idx2) {
 
     int s_len = strlen(s);
     if(idx1 > s_len || idx1 <= (-1*s_len)){
-        printf("String index out of Bounds\n");
+        printf("String index out of bounds\n");
```

Comment:
I think `SemanticError` could be thrown either at all places or at none else it won't really remove the inconsistency completely. Throwing at all places is better I guess. Thanks !

Reply:
I would say we can stick to returning `SemanticError` in the `python_ast_to_asr.cpp` file and use print statements otherwise like it is being done !